src/test/java/mate.academy/security/AuthenticationServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package mate.academy.security;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import mate.academy.service.RoleService;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+class AuthenticationServiceImplTest {
+    private static final String EMAIL = "sobaka@mail.ru";
+    private static final String  PASSWORD = "12345";
+    private static final String NON_EXISTING_EMAIL = "neo@mail.com";
+    private static final String  NON_EXISTING_PASSWORD = "777";
+    private static AuthenticationService authenticationService;
+    private static UserService userService;
+    private static RoleService roleService;
+    private static PasswordEncoder passwordEncoder;
+    private static User user;
+    private static User non_existing_user;
```

Comment:
```suggestion
    private static User nonExistingUser;
```
use camelCase naming